src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, String color) {
+        super(String.valueOf(color));
```

Comment:
Passing 'String.valueOf(color)' is redundant since 'color' is already a String. You can pass 'color' directly to the superclass constructor.